brayns/common/scene/Model.h

```diff
@@ -97,9 +97,12 @@ class ModelParams : public ModelInstance
     const std::string& getName() const { return _name; }
     void setPath(const std::string& path) { _updateValue(_path, path); }
     const std::string& getPath() const { return _path; }
+    void setBVHType(BVHType bvhType) { _updateValue(_bvhType, bvhType); }
+    BVHType getBVHType() const { return _bvhType; }
```

Comment:
It would be better if this parameter goes into the ModelDescriptor property map because then you can set it from the client. I'm not totally sure about the jsonrpc part, maybe @rolandjitsu can comment on this.

Reply:
If you add it to the property map it would be rendered without any effort on my side (because it's generated from schema).

But then you have to make sure that when the bvh property on the property map is set, you do your stuff (set the bvh type).

Also, I'm not sure how enums are handled currently for the property map props (if they are).

Reply:
In fact it's not an enum, it's a bit mask. From the command line I think it's set as a space separated list of strings. I'd guess you don't have any code for rendering such a thing.

Reply:
Nope, we don't handle bit masks.

For multiple selection we'd need some additional info from schema (instead of `{type: 'string', enum: ['bla', ...]}` has to be `{type: 'array', items: {type: 'string', enum: ['bla', ...]}`).

And then I'd need to add support for multiple selection.

If this is single selection, then we just need string enums instead of bitmask.

Reply:
@tribal-tec requested it to be a multiple selection, but not all combinations are allowed... Another option is splitting this into multiple properties:
- BVH update hint: static|dynamic
- compact BVH: bool
- robust BVH: bool
What do you think @karjonas ?

Reply:
This should be easy to render and we only need to extend the schema to support single selection for a set of values.

Reply:
As discussed offline, let's keep it as it is for the moment and open this can of worms later.